vulnerabilities/helpers.py

```diff
@@ -267,3 +267,16 @@ def _get_gh_response(gh_token, graphql_query):
     endpoint = "https://api.github.com/graphql"
     headers = {"Authorization": f"bearer {gh_token}"}
     return requests.post(endpoint, headers=headers, json=graphql_query).json()
+
+
+def get_unique_patched_packages_with_affected_packages(
+    affected_packages: List[AffectedPackage],
+):
+    unique_patched_packages_with_affected_packages = {}
```

Comment:
Using a defaultdict is cleaner IMHO for this. Also you may want to use slightly shorter variable names:
```suggestion
    unique_patched_packages_with_affected_packages = defaultdict(list)
```